src/react/components/APILink.js

```diff
@@ -1,45 +1,93 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import qs from 'querystringify';
+import { connect } from 'react-redux';
 import './APILink.scss';
 
 const propTypes = {
   disabled: PropTypes.bool,
   isCsLink: PropTypes.bool,
   name: PropTypes.string,
+  showNewWindow: PropTypes.bool,
   title: PropTypes.string,
   ucFrom: PropTypes.string,
   ucFromLink: PropTypes.string,
   ucFromText: PropTypes.string,
+  ucUpdateCache: PropTypes.string,
   url: PropTypes.string,
   urlId: PropTypes.string,
 };
 
-const APILink = ({ disabled, title, name, url, ucFrom, ucFromText }) => {
+const APILink = ({
+  disabled,
+  showNewWindow,
+  title,
+  name,
+  url,
+  ucFromLink,
+  ucFromText,
+  ucUpdateCache,
+}) => {
+  if (!url) {
+    // TODO - how to handle when Link API is not coming from CS?
+    // eslint-disable-next-line no-console
+    console.log(
+      '**URL not found** Component could not be rendered.  Check Link API data.'
+    );
```

Comment:
Let's remove this console statement and comment, just returning null if no URL is present.

Reply:
Thanks